index.md

```diff
@@ -12,3 +19,4 @@
 1. ОКНА
 1. ВОДУ
 1. ТУАЛЕТ
+1. ПОДСОБКУ (ЗИМОЙ)
```

Comment:
We should follow the same style for all entries; since we have no seasonal conditions for air conditioning system, we should omit it here too.